os/arch/arm/src/armv7-m/up_lazyexception.S

```diff
@@ -210,7 +210,7 @@ exception_common:
 	 * and call up_doirq to handle the interrupt
 	 */
 
-	cpsid	i						/* Disable further interrupts */
+	/*cpsid   i */				/* Disable further interrupts */
```

Comment:
can you use proper format or combine these two in one comment

Reply:
modified the code to make it macro protected